### PR TITLE
fix: update v5 wss public ping heartbeat

### PR DESCRIPTION
**Pull request by josephyim224**

### CODE REVIEW

**Comment by josephyim224** [comment]
ref: https://github.com/hirokisan/bybit/blob/5391e166586de682e940dbb9beb925d898b02974/v5_ws_private.go#L232
align the heartbeat message in public wss to the one in private

File: v5_ws_public.go [hunk 1]
```diff
@@ -234,7 +234,7 @@ func (s *V5WebsocketPublicService) Run() error {
 
 // Ping :
 func (s *V5WebsocketPublicService) Ping() error {
-	if err := s.writeMessage(websocket.PingMessage, nil); err != nil {
+	if err := s.writeMessage(websocket.TextMessage, []byte(`{"op":"ping"}`)); err != nil {
```
Comment (hirokisan) [inline v5_ws_public.go:237]: ref: https://bybit-exchange.github.io/docs/v5/ws/connect#how-to-send-the-heartbeat-packet

スクリーンショット 2023-05-06 9 52 40

Comment (hirokisan) [reply v5_ws_public.go:237]: wss private is same as above

https://github.com/hirokisan/bybit/blob/5391e166586de682e940dbb9beb925d898b02974/v5_ws_private.go#L232-L234

**Review by hirokisan — APPROVED** [review]
Thank you for your contribution!

LGTM 👍

**Pull request merged**